### DISCUSSION TITLE
Line spacing of bulleted (unordered) lists

### DISCUSSION THREAD

**Discussion by PeterParslow** [agiorguk/gemini#92] [General]
In the HTML output, each list item is unnecessarily placed inside a <p> element which has the result of spreading the bulleted list items out rather. E.g. in https://agiorguk.github.io/gemini-dev/1048-uk-gemini-encoding-guidance.html you can see:

```
<div class="ulist">
<ul>
<li>
<p>The XML declaration</p>
</li>
<li>
<p>XML namespace identifiers</p>
</li>
</ul>
</div>
```
I am pretty sure this would be more visually presentable as 

```
<div class="ulist">
<ul>
<li>The XML declaration</li>
<li>XML namespace identifiers</li>
</ul>
</div>
```
[On the live page (AGI site), it is not in a div class, and the li elements are indented]

Can that be controlled in the way that the HTML is generated from the ASCIIDOC file?
Note: the bulleted list items are also quite spaced out in the ASCIIDOC preview on GitHub

**Comment by archaeogeek** [comment]
This looks to be quite a contentious issue that has been around for some time- the short version is that it's by design, and github is different because of the way it renders asciidoc compared to the actual converter. There's a potential fix in the comments that follow on from https://github.com/asciidoctor/asciidoctor/issues/1087#issuecomment-222332790 that I could play around with though

**Comment by archaeogeek** [comment]
I've got a fix for this:
Before:
![Screenshot from 2023-03-27 15-10-47](https://user-images.githubusercontent.com/1544849/227965208-ec0e2d57-9b0c-42a1-9ae9-0b211cd6e486.png)

After:
![Screenshot from 2023-03-27 15-10-25](https://user-images.githubusercontent.com/1544849/227965250-9916f6cf-0d40-48f6-8405-6775e7400bd1.png)

It uses the method in the issue I linked to- but with the correctly named file (ulist.html.slim, not ulist.slim.html) and then the addition of the following parameters in the asciidoctor command:
`-T templates -E slim`

Are we happy with this as a fix? If so I'll adjust the github actions to include the additional parameters

**Reply by PeterParslow** [reply]
Looks great to me; thanks for poking around to find it!
